projects/novo-elements/src/elements/data-table/data-table-clear-button.component.ts

```diff
@@ -44,31 +44,43 @@ export class NovoDataTableClearButton<T> {
   queryClear: EventEmitter<boolean> = new EventEmitter();
   @Output()
   allClear: EventEmitter<boolean> = new EventEmitter();
+  @Input()
```

Comment:
For any new boolean inputs, could you add the BooleanInput annotation? That allows simpler HTML expression of a boolean eg;

`<novo-data-table-clear-button emitOnly>`

Reply:
done